open_ce/errors.py

```diff
@@ -89,6 +89,9 @@ class Error(Enum):
     TEMP_BUILD_IMAGE_FILES = (37, "Error removing temporary files created during build image.")
     UNABLE_DOWNLOAD_SOURCE = (38, "Unable to download source for '{}'.")
     UNABLE_CLONE_SOURCE = (39, "Unable to clone source for '{}'.")
+    GCC10_11_COMPILER_NOT_FOUND = (40, "GCC10 and 11 are either not installed or are installed at non-default location." +
```

Comment:
I think we can split this into two error codes.

Reply:
I think we can reword this error message but there is no need to have two separate error codes for GCC 10 and GCC 11. I'll reword the message.

Reply:
Okay, thats fine too.